2019-03-Autumn/3_stacks_queues/README.md

```diff
@@ -36,41 +36,59 @@ If the problem is simple enough, remove this section. TODO :bug:
 
 TODO :bug:
 
-### 3. PROBLEM 3 TODO :bug:
+### 3. Reverse Substrings In Parentheses
 
-Source: TODO :bug:
+Source: [LeetCode](https://leetcode.com/problems/reverse-substrings-between-each-pair-of-parentheses/) 
 
 #### Scenario
 
-Problem Statement TODO :bug:
+You are given a string `s` that consists of lower case English letters 
+and brackets. 
 
-#### Example Input
+Reverse the strings in each pair of matching parentheses, starting 
+from the innermost one.
 
-If the problem is simple enough, remove this section. TODO :bug:
+Your result should not contain any brackets.
 
-#### Function Signature
+<a name="constraints">
 
-TODO :bug:
+Constraints:
 
-## Solutions
+* `0 <= s.length <= 2000`
+* `s` only contains lower case English characters and parentheses.
```

Comment:
The first 2 constraints don't really matter for the problem imo

Reply:
I agree; I am inclined to leave the second constraint because I think it will help people focus on core implementation requirements.